app/graphql/mutations/users/update_user_details_mutation.rb

```diff
@@ -0,0 +1,20 @@
+module Mutations
+  module Users
+    class UpdateUserDetailsMutation < Mutations::BaseMutation
+      description 'Updates a User with the access_token'
+      null true
+      argument :accessToken, String, required: false
+      argument :githubUsername, String, required: false
+      argument :githubOrg, String, required: false
+      field :errors, [String], null: true
+      def resolve(data)
+        current_user = context[:current_user]
+        if data[:github_org]
+          current_user.organization_id = Organization.find_or_create_by(name: data[:github_org]).id
```

Comment:
This would have to be a:

```ruby
Organization.find_by(name: data[:github_org])
```

with a subsequent `raise`, should it return `nil`. 

Reason being, we can't make any call's to GitHub's API to fetch the user's data, unless the `Organization` record associated with the user is both:

- a valid organization on GitHub
- an organization that the user's `GithubUser` is actually associated with in GitHub

Reply:
yeah thats true, once its a `<select>` it takes care of itself completely.
I'll put a few in the `seeds` file and maybe long term hit the API for a lookup on the user's orgs?